scss/main.scss

```diff
@@ -890,7 +890,8 @@ $animation-length: 80s;
 }
 
 .link, .link:visited {
-  color: black;
+  font-weight: 300;
+  color: #A88AA8;
```

Comment:
hmm this changes all the links to purple, which includes some links in the registration/tracks section. what you can do is either 
1. leave this as is, and then create a separate black-link class add add that to the registration/tracks links
2. change this back to black, and then create a separate purple-link class for all the faq links

option 1 is easier imo since theres only two links that should be black

some more nit picks:
- use $lavender instead of hardcoding the hexcode
- keep the links bold 
- add the purple color for link:hover so it doesnt turn back to black on hover